libs/core/src/components/_internal/pds-label/pds-label.scss

```diff
@@ -1,19 +1,14 @@
-// TODO: move these styles into base/reset files
 :host {
   display: inline-block;
 }
 
 label, pds-label {
-  --color-text-default: var(--pine-color-grey-900);
-
-  --font-family-default: var(--pine-font-family-circular);
-
-  --font-weight-default: var(--pine-font-weight-medium);
-
-  --letter-spacing-default: var(--pine-letter-spacing-xs);
+  --color-text-default: var(--pine-color-grey-950);
+  --color-text-disabled: var(--pine-color-grey-600);
 
   color: var(--color-text-default);
-  font-family: var(--font-family-default);
-  font-weight: var(--font-weight-default);
-  letter-spacing: var(--letter-spacing-default);
+
+  [disabled] &, *:disabled & {
+    color: var(--color-text-disabled);
+  }
 }
```

Comment:
nitpick, missing new line. checkout https://marketplace.visualstudio.com/items?itemName=jdforsythe.add-new-line-to-files if you're using vscode.

Reply:
Thanks for this, installing now!